Vagrantfile

```diff
@@ -12,14 +14,36 @@ override_hash = nil
 # Check and override if exist any match JSON object from vagrant_config_override.json
 if File.exist? (File.dirname(__FILE__) + '/vagrant_config_override.json')
   override_file = File.read(File.dirname(__FILE__) + '/vagrant_config_override.json')
-
+  parsing_file = File.dirname(__FILE__) + '/vagrant_config_override.json'
   begin
     override_hash = JSON.parse(override_file)
+
     data_hash = overrides(data_hash, override_hash)
+
+    if data_hash.key?('config_paths')
+      data_hash['config_paths'].each do |default_config_file_path, value|
+        overide_config_file_path = default_config_file_path.gsub(/default\.json$/, "overide.json")
+        if File.exist? (default_config_file_path)
```

Comment:
if the default config file does not exist => print the error message + exit immediately. This is considered a fatal error to be fixed by users immediately.